bootstrap/api/endpoint_test.go

```diff
@@ -160,7 +160,7 @@ func dec(in []byte) ([]byte, error) {
 func newService(auth mainflux.AuthServiceClient, url string) bootstrap.Service {
 	things := mocks.NewConfigsRepository()
 	config := mfsdk.Config{
-		BaseURL: url,
+		BootstrapURL: url,
```

Comment:
you dont have ThingsURL here

Reply:
Why do we need it and where is defined? In an ENV var?